src/auth/auth-api-request.ts

```diff
@@ -1323,3 +1368,49 @@ export class FirebaseAuthRequestHandler {
       });
   }
 }
+
+
+/**
+ * Utility for sending requests to Auth server that are Auth instance related. This includes user and
+ * tenant management related APIs. This extends the BaseFirebaseAuthRequestHandler class and defines
+ * additional tenant management related APIs.
+ */
+export class FirebaseAuthRequestHandler extends BaseFirebaseAuthRequestHandler {
```

Comment:
My suggestions for names would be `AuthRequestHandler` and `TenantAwareAuthRequestHandler`.

Reply:
Done